sklearn_pandas/features_generator.py

```diff
@@ -0,0 +1,50 @@
+def gen_features(columns, classes=None):
+    """Generates a feature definition list which can be passed
+    into DataFrameMapper
+
+    Params:
+
+    columns     a list of column names to generate features for.
+
+    classes     a list of classes for each feature, a list dictionaries with
+                transformer class and init parameters, or None.
+
+                If list of classes is provided, then each of them is
+                instantiated with default arguments:
```

Comment:
I'd write "with default arguments. Example:". Otherwise it is a bit confusing.

Reply:
Got it.